pageserver/src/tenant/timeline/compaction.rs

```diff
@@ -303,6 +363,10 @@ impl Timeline {
             remote_client.wait_completion().await?;
         }
 
+        fail::fail_point!("compact-shard-ancestors-persistent-kill", |_| {
+            std::process::abort()
+        });
```

Comment:
We already have the `exit` action.

Reply:
Huh, how did I miss that?  changed these to use 'exit' in f28f0ce36